oraichain/chain.json

```diff
@@ -34,47 +34,67 @@
   },
   "codebase": {
     "git_repo": "https://github.com/oraichain/orai",
-    "recommended_version": "v0.41.7",
+    "recommended_version": "v0.42.4",
     "compatible_versions": [
-      "v0.41.3",
-      "v0.41.4",
-      "v0.41.6",
-      "v0.41.7"
+      "v0.42.4"
     ],
-    "cosmwasm_version": "0.30.2",
+    "cosmos_sdk_version": "v0.45.16",
+    "cosmwasm_version": "0.33.0",
     "cosmwasm_enabled": true,
     "genesis": {
       "genesis_url": "https://raw.githubusercontent.com/oraichain/oraichain-static-files/master/genesis.json"
     },
     "versions": [
       {
         "name": "v0.40",
+        "height": 2098755,
+        "proposal": 32,
         "recommended_version": "v0.40.3",
         "compatible_versions": [
+          "v0.40.0",
+          "v0.40.1",
+          "v0.40.2",
           "v0.40.3"
         ],
-        "binaries": {
-          "linux/amd64": "https://orai.s3.us-east-2.amazonaws.com/v0.40.3/oraid"
-        },
         "next_version_name": "v0.41"
       },
       {
         "name": "v0.41",
-        "height": 12353514,
-        "proposal": 185,
-        "recommended_version": "v0.41.7",
+        "height": 9058418,
+        "proposal": 116,
+        "recommended_version": "v0.41.8",
         "compatible_versions": [
+          "v0.41.0",
+          "v0.41.1",
+          "v0.41.2",
           "v0.41.3",
           "v0.41.4",
           "v0.41.5",
-          "v0.41.7"
+          "v0.41.6",
+          "v0.41.7",
+          "v0.41.0"
```

Comment:
v.0.41.0 is repeated

Reply:
You're right; it's my fault. I updated to v0.41.8. Thanks.